src/main/kotlin/app/revanced/utils/adb/Adb.kt

```diff
@@ -61,6 +61,8 @@ internal class Adb(
             // relaunch app
             device.run(Constants.COMMAND_RESTART.replacePlaceholder())
 
+            logger.info("Started logging app")
```

Comment:
```suggestion
            logger.info("Logging")
```

Reply:
Isn't this a little arbitrary?

Reply:
In that case the line can be removed as logging starts right away.

Reply:
Alright.